src/variables.scss

```diff
@@ -657,7 +657,7 @@ $modal-content-border-color:                   rgba(0,0,0,.2);
 $modal-content-fallback-border-color:          #999;
 
 //** Modal backdrop background color
-$modal-backdrop-bg:           #000;
+$modal-backdrop-bg:           rgba(0,0,0,.5);
```

Comment:
The opacity is applied automatically through the next settings for normal backdrops. So you can revert this if you set a backdrop as described above.

Reply:
Reverted.